### PR TITLE
VIH-6446 link defence advocate and video access point

### PR DESCRIPTION
**Pull request by RocheL**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-6446
### Change description ###
 - updated ui for defence advocate
**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: AdminWebsite/AdminWebsite.UnitTests/Controllers/HearingsController/EditHearingTests.cs [hunk 1]
```diff
@@ -458,7 +458,7 @@ public async Task Should_remove_endpoint_if_endpoint_is_removed_from_the_endpoin
             ((OkObjectResult)result.Result).StatusCode.Should().Be(200);
 
             _bookingsApiClient.Verify(x => x.RemoveEndPointFromHearingAsync(It.IsAny<Guid>(), It.IsAny<Guid>()), Times.Once);
-            _bookingsApiClient.Verify(x => x.UpdateDisplayNameForEndpointAsync(It.IsAny<Guid>(), It.IsAny<Guid>(), It.IsAny<UpdateEndpointRequest>()), Times.Once);
+            _bookingsApiClient.Verify(x => x.UpdateDisplayNameForEndpointAsync(It.IsAny<Guid>(), It.IsAny<Guid>(), It.IsAny<UpdateEndpointRequest>()), Times.Exactly(2));
```
Comment (shaed-parkar) [inline AdminWebsite/AdminWebsite.UnitTests/Controllers/HearingsController/EditHearingTests.cs:461]: Why should the update be called on a remove?

**Review by shaed-parkar — COMMENTED** [review]
Just a concern regarding tests around defence advocates and why we need to call update endpoint when removing them

File: AdminWebsite/AdminWebsite.UnitTests/Controllers/HearingsController/EditHearingTests.cs (hunk 1 above)
Comment (RocheL) [reply AdminWebsite/AdminWebsite.UnitTests/Controllers/HearingsController/EditHearingTests.cs]: separated the test

**Review by shaed-parkar — APPROVED** [review]
Thanks Lionel. It's always best to create new tests for new functionality rather than re-use existing tests so it's clear what the scenario under test is

**Pull request merged**